spec/requests/posts_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe 'Posts', type: :request do
   describe 'GET #index' do
     before(:each) do
-      get user_posts_path(user_id: 1)
+      get user_posts_path(user_id: 3)
```

Comment:
- [ ] Please fix 2tests that are failing
![fail](https://user-images.githubusercontent.com/44745855/167168063-63233cd4-a9ac-45db-aa1a-aa73839a08c6.png)

Reply:
or run the seed file
will give you user id :1 
and post id :1